.gitignore

```diff
@@ -1,9 +1,9 @@
 build
-
 cmake-build-release
 
 ## idea
 cmake-build-debug
+cmake-build-debug-event-trace
```

Comment:
I think we use `cmake-build-*` to replace all three entries that fall under regex.